### PR TITLE
Actually dump chunk packet for 1.18

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by nickelpro** [comment]
Don't dump light packets independently for 1.18, they're meaningless.

Add something like this to the constructor
```js
this.withLightPackets = this.mcData.isNewerOrEqualTo('1.14') && this.mcData.isOlderThan('1.18')
```

**Comment by rom1504** [comment]
> Don't dump light packets independently for 1.18, they're meaningless.

Can you explain why ?

**Comment by nickelpro** [comment]
Sorry, meaningless is the wrong word. In 1.18 all fields from the Light Update packet are included in the initial Chunk Data packet, and no Light Update packet is sent unless there's some change to the lighting.

What this means is you can't reliably dump Light Update packets, you would need special test code in pris-chunk to only test the ones that were incidentally dumped and the testing doesn't do anything for you. The same `loadParsedLight` routine is used for both the initial Chunk Data packet and the Light Update packet since the format of the lighting fields is identical.

Basically we shouldn't dump Light Updates in 1.18+ for the same reason we don't dump Multi-Block Change packets or Tile Entity Data packets: they can't be reliably generated with just chunk-dumper and the data they provide for testing is redundant.

**Comment by rom1504** [comment]
ok but then the code should be adapted to do that
in the current state it will save light packets

**Comment by rom1504** [comment]
merged nickelpro one
